src/components/Metrics.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Metrics = () => {
+  const chosenCoin = useSelector((state) => state.metrics.data);
+
+  if (!chosenCoin) {
+    return (
+      <div className="loading-container">
+        <div className="spinner-border text-secondary" role="status">
+          <span className="visually-hidden">Loading...</span>
+        </div>
+        <div>Loading...</div>
+      </div>
+    );
+  }
+
+  return (
+    <div className="coin metrics">
+      <div className="coin-symbol">{chosenCoin.coinSymbol}</div>
+      <img src={chosenCoin.coinImage} alt={chosenCoin.coinName} />
+      <div className="coin-name">{chosenCoin.coinName}</div>
+      <div className="coin-price">
+        Current Price:&nbsp;
+        {chosenCoin.coinPrice}
+      </div>
+    </div>
```

Comment:
- [ ] I checked the API you used and noticed that it has lots of metrics data that you could have shown on the details page just as how the original design template shows. The motive of this project is about building a web mobile app that displays metric data about a topic of your choice. Kindly display more data and please make your design match the original design when displaying metric data on the details page. 👍🏽

**API Data returned**

<img width="851" alt="Screenshot 2023-06-10 at 7 57 49 PM" src="https://github.com/CarlosIgreda/React-Capstone/assets/50941074/075cc85a-9967-44fa-af15-ca9f9c43d12a">

**Your Design**

<img width="1318" alt="Screenshot 2023-06-10 at 7 56 13 PM" src="https://github.com/CarlosIgreda/React-Capstone/assets/50941074/f5619d3a-9266-4756-9086-aec3f30d3da8">

**The Original Design**

<img width="388" alt="Screenshot 2023-06-10 at 8 12 30 PM" src="https://github.com/CarlosIgreda/React-Capstone/assets/50941074/4b2b276e-2242-4b27-8353-4ac0ed392d72">

Reply:
Done it.